plugins/nf-tower/src/main/io/seqera/tower/plugin/CacheManager.groovy

```diff
@@ -104,22 +107,6 @@ class CacheManager {
         if( !remoteWorkDir || !sessionUuid )
             return
 
-        if( !Files.exists(localCachePath) ) {
-            log.debug "Local cache path does not exist: $localCachePath — skipping cache backup"
-            return
-        }
-
-        // upload nextflow cache metadata
-        try {
-            log.info "Saving cache: ${localCachePath.toUriString()} => ${remoteCachePath.toUriString()}"
-            remoteCachePath.deleteDir()
-            remoteCachePath.parent.mkdirs()
-            FilesEx.copyTo(localCachePath, remoteCachePath)
-        }
-        catch (Throwable e) {
-            log.warn "Failed to backup resume metadata to remote store path: ${remoteCachePath.toUriString()} — cause: ${e}", e
-        }
```

Comment:
Is not enough wrapping this into a `if( Files.exists(localCachePath) ) {`

Reply:
It is just wrapping if we do not want to warn/log when localCachePath doesn't exist. If you prefer to do not move the code I can just implement it with an if-else.

Reply:
changed